src/Models/ContactResponse.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tipoff\Forms\Models;
+
+use Illuminate\Database\Eloquent\SoftDeletes;
+use Tipoff\Support\Models\BaseModel;
+use Tipoff\Support\Traits\HasPackageFactory;
+
+class ContactResponse extends BaseModel
+{
+    use HasPackageFactory;
+    use SoftDeletes;
+
+    public $timestamps = false;
```

Comment:
Should this be used if both timestamps are excluded or should I stick to using `CONST` for disabling the default timestamps?

Reply:
I think this is the correct way if both timestamps are absent.

Let's add the created_at timestamp to this model though.

Reply:
https://github.com/tipoff/forms/pull/43/commits/4ea1d6d480878eb39341bb25644597f4ae9cf49c